src/components/EditPasswordForm/inplayer/EditPasswordForm.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import styles from '../EditPasswordForm.module.scss';
+
+import type { FormErrors } from '#types/form';
+import type { EditPasswordFormData } from '#types/account';
+import FormFeedback from '#components/FormFeedback/FormFeedback';
+import TextField from '#components/TextField/TextField';
+import Button from '#components/Button/Button';
+import LoadingOverlay from '#components/LoadingOverlay/LoadingOverlay';
+import { testId } from '#src/utils/common';
+import PasswordField from '#src/components/PasswordField/PasswordField';
+import { useAccountStore } from '#src/stores/AccountStore';
+
+type Props = {
+  onSubmit: React.FormEventHandler<HTMLFormElement>;
+  onChange: React.ChangeEventHandler<HTMLInputElement | HTMLTextAreaElement>;
+  onBlur: React.FocusEventHandler<HTMLInputElement | HTMLTextAreaElement>;
+  error?: string;
+  errors: FormErrors<EditPasswordFormData>;
+  value: EditPasswordFormData;
+  submitting: boolean;
+};
+
+const EditPasswordForm: React.FC<Props> = ({ onSubmit, onChange, onBlur, value, errors, submitting }: Props) => {
+  const { t } = useTranslation('account');
+  const user = useAccountStore.getState().user;
+
+  return (
+    <form onSubmit={onSubmit} data-testid={testId('change-password-form')} noValidate className={styles.forgotPasswordForm}>
+      <h2 className={styles.title}>{t('reset.password_reset')}</h2>
+      {errors.form ? <FormFeedback variant="error">{errors.form}</FormFeedback> : null}
+      {!user ? (
+        <TextField
+          className={styles.textField}
+          value={value.resetPasswordToken}
+          onChange={onChange}
+          onBlur={onBlur}
+          label={t('reset.reset_password_token')}
```

Comment:
Where does this token come from? Is it sent in the password reset email?  Does it need to be visible to the user?

Reply:
Yes, it is sent in the password reset email and looks like this: `325a2693f931750a`, and yes, it should be visible to the users.